packages/carbon-addons-boomerang-react/src/components/FeatureNavTab/_featureNavTab.scss

```diff
@@ -18,5 +18,8 @@
     border-bottom: 0.125rem solid $support-02;
     color: $text-01;
     font-weight: 600;
+    &:hover {
```

Comment:
@LucasGrimauth good idea adding the hover state here. I think we should update this to have the same transition animation as the UIShell nav items

Reply:
Done